MerlinAU.sh

```diff
@@ -3677,7 +3671,7 @@ Please manually update to version $minimum_supported_version or higher to use th
 
         if [ ! -f "$changeLogFile" ]
         then
-            Say "Change-log file [$changeLogFile] does NOT exist."
+            Say "Change-log file [${FW_BIN_DIR}/$Changelog-${changeLogTag}.txt] does NOT exist."
```

Comment:
Approved. Smart call I actually forgot about this from his screenshot but it said blank file was missing lol!